OpenTelemetry.sln

```diff
@@ -16,6 +16,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		NuGet.config = NuGet.config
 		OpenTelemetry.proj = OpenTelemetry.proj
 		README.md = README.md
+		Releasing.md = Releasing.md
```

Comment:
Consider making this uppercase:
1. align with other top level *.md files (README, CONTRIBUTING)
2. align with OpenTelemetry Python https://github.com/open-telemetry/opentelemetry-python/blob/master/RELEASING.md